app/src/Event/EventDb.php

```diff
@@ -2,10 +2,15 @@
 namespace Event;
 
 use Application\BaseDb;
+use Application\CacheService;
 
 class EventDb extends BaseDb
 {
-    protected $keyName = 'events';
+    public function __construct(CacheService $cache)
```

Comment:
`protected $keyName` is now in the base class. This change uses the constructor to set its value rather than setting the value outside of the constructor.